### PR TITLE
Monkey butt2

### PR DESCRIPTION
**Pull request by dustinmcintosh**
Fix a bunch of math and tables

### CODE REVIEW

File: content/articles/methods/metropolis.md [hunk 1]
```diff
@@ -46,69 +46,69 @@ Metropolis is an iterative, try-accept algorithm. We initialize the algorithm by
...
 total_samples = 200000
 
-# Function used to decide move acceptance  
-def posterior_numerator(mu):  
-prod = 1  
-for x in X:  
-prod *= np.exp(-(x - mu) ** 2 / 2)  
+# Function used to decide move acceptance
+def posterior_numerator(mu):
+prod = 1
```
Comment (frangipane) [inline content/articles/methods/metropolis.md:83]: indentation unfortunately messed up for most code blocks :(

File: content/articles/methods/gpu-accelerated-theano-keras-with-windows-10.md [hunk 2]
```diff
@@ -77,28 +76,28 @@ Testing Theano with GPU
...
-if numpy.any([isinstance(x.op, T.Elemwise) for x in f.maker.fgraph.toposort()]):  
-print('Used the cpu')  
-else:  
+rng = numpy.random.RandomState(22)
+x = shared(numpy.asarray(rng.rand(vlen), config.floatX))
+f = function([], T.exp(x))
+print(f.maker.fgraph.toposort())
+t0 = time.time()
+for i in range(iters):
+r = f()
```
Comment (frangipane) [inline content/articles/methods/gpu-accelerated-theano-keras-with-windows-10.md:94]: code indentation

File: content/articles/methods/model-auc-depends-on-test-set-difficulty.md [hunk 3]
```diff
@@ -54,26 +55,26 @@ Notice that if a test example has a mean very close to zero, it will be difficul
...
-ax1.plot(fpr, fpr, 'k--')  
-ax2.plot(means, 0 * means, '*', label=means_std)  
-ax1.legend(loc='lower right', shadow=True)  
-ax2.legend(loc='lower right', shadow=True)  
-ax1.set_title('TPR versus FPR -- The ROC curve')  
-ax2.set_title('Means sampled for each case')  
+SAMPLES = 1000
+means_std = 0.1
+for means_std in [3, 0.5, .001]:
+means = means_std * np.random.randn(SAMPLES)
```
Comment (frangipane) [inline content/articles/methods/model-auc-depends-on-test-set-difficulty.md:67]: indentation

File: content/articles/statistics/model-selection.md [hunk 4]
```diff
@@ -24,60 +24,60 @@ We now walk through our two examples.
...
+$$ \tag{7}
+\hat{\theta} = \begin{cases}
+\theta_0 - \Lambda \sigma^2 & \text{if } \theta_0 - \Lambda \sigma^2 > 0 \\
+0 & \text{else}.
+\end{cases}
+$$
+The mean squared error of the estimator is
+$$ \tag{8}
+\langle (\hat{\theta} - \theta_*)^2 \rangle \equiv \int \frac{1}{\sqrt{2 \pi \sigma^2}}
+e^{ - \frac{(\theta_0 - \theta_*)^2}{2 \sigma^2}} \left ( \hat{\theta} - \theta_* \right)^2 d \theta_0.
```
Comment (frangipane) [inline content/articles/statistics/model-selection.md:67]: seems better to keep as `exp` given how complicated the argument is, i.e. a fraction.  just my opinion tho

File: content/articles/methods/battleship.md [hunk 5]
```diff
@@ -91,123 +91,123 @@ That's it for the basics of deep, policy-gradient RL. We now turn to our python
...
-bomb_index = np.argmax(probs)  
-# update board, logs  
-hit_log.append(1 * (bomb_index in ship_positions))  
-current_board[0][bomb_index] = 1 * (bomb_index in ship_positions)  
-action_log.append(bomb_index)  
-return board_position_log, action_log, hit_log  
+```
+TRAINING = True
+def play_game(training=TRAINING):
+""" Play game of battleship using network."""
```
Comment (frangipane) [inline content/articles/methods/battleship.md:139]: indentation

File: content/articles/statistics/bonferroni-correction-for-multiple-pairwise-comparison-tests.md [hunk 6]
```diff
@@ -36,171 +36,85 @@ The reason why we refer to (\ref{bon2}) as the naive Bonferroni approximation is
...
-$$  
-This is interesting because the individual pairwise tests have p-values given by  
-$$ \tag{7} \label{asy_i}  
-p_i = \int_{-k\sqrt{2}\sigma}^{k\sqrt{2}\sigma} \frac{e^{-x^2 / (4 \sigma^2)}}{\sqrt{4 \pi \sigma^2 }} = \text{erf}(k /\sqrt{2}) \sim 1 - \frac{e^{-k^2/2}}{k \sqrt{\pi/2}}.  
-$$  
-At large $k$, this is dominated by the exponential. Comparing with (\ref{asy_g}), this implies  
-$$ \tag{8} \label{fin_g}  
-p_f \sim \left (1 - \alpha_i^{1/2} \right)^N \sim 1 - N \alpha_i^{1/2} \equiv 1 - \alpha_f.  
-$$  
+| $N$ | $p_{f,exact}$ |$k_{exact}$ | $k_{Bon}$ | $p_{f, Bon}$ |
```
Comment (frangipane) [inline content/articles/statistics/bonferroni-correction-for-multiple-pairwise-comparison-tests.md:57]: oooh nice job fixin up this table

**Review by frangipane — APPROVED** [review]
there's more stuff that needs to be fixed but up to you how much you want to do before you merge this since it's a lot

File: content/articles/statistics/model-selection.md (hunk 4 above)
Comment (dustinmcintosh) [reply content/articles/statistics/model-selection.md:67]: absoultely not

**Pull request merged**